### DISCUSSION TITLE
Text component does not honour the selected theme

### DISCUSSION THREAD

**Discussion by aflatoon2874** [prismaneui/prismane#97] [General]
Hello team,

Just wondering why the `Text` component does not follow the selected theme. On a light theme the text gets displayed properly with the base color and 500 weight but on the dark theme it does not adapt to the theme, it remains dark whereas other components such as `TextField`, `Button` do follow the theme automatically.

Similarly, the `Center` component also does not adapt the background color as per the theme.

**Comment by spleafy** [comment]
Hi, you are right, the `Text` component was left without any color and inherits the color from its parent or the document. This should however be improved by adding colors and making it adapt to the theme.

Do you think that it is a good idea to make a few sets of colors based on the value of the `as` prop? For example, a lighter color if the text is a `p` or a `span` tag or just a single color for all elements.

Just to note, the `Center` component only centers its content and does not provide any sort of color background by default.

Thank you for reporting this, it surely will be added in the next minor version (v1.5.0)!

**Reply by aflatoon2874** [reply]
Hi, thanks for a prompt reply. I think a single color for all elements should do for light and dark theme. If color variation is needed then it should be manually overridden and adjusted for light and dark theme.

**Reply by spleafy** [reply]
Thank you for the fast reply! Thank you so much for helping and giving your opinion! I will make sure that this feature gets added in the new minor version as well as different font weights for headings and predefined font sizes for different elements. Will update this discussion when the new version is released.

If you want to get faster replies, more in-depth conversations, and day-to-day updates on Prismane, you can join our [Discord Server](https://discord.gg/gFvcmdpKeb)

**Comment by LuuKhoaHoc** [comment]
> Hello team,
> 
> Just wondering why the `Text` component does not follow the selected theme. On a light theme the text gets displayed properly with the base color and 500 weight but on the dark theme it does not adapt to the theme, it remains dark whereas other components such as `TextField`, `Button` do follow the theme automatically.
> 
> Similarly, the `Center` component also does not adapt the background color as per the theme.

Can you give more example?, Im using and did not recognize it

**Reply by aflatoon2874** [reply]
Watch the official `Prismane in action` video. Martin has amply highlighted this problem and to fix it manually adjusted the colors.

**Reply by LuuKhoaHoc** [reply]
Oh, i saw, i not really good at English and did not recognize it rerported at Prismane video. Thank you dude

**Comment by spleafy** [comment]
Hi, the abovementioned feature has just been introduced in the new minor version of Prismane ([v1.5.0](https://github.com/prismaneui/prismane/releases/tag/v1.5.0)). 

Please, check the release notes, to learn about the new features the version introduced and all the improvements and bug fixes.
